### ISSUE TITLE
Please save volume levels

### ISSUE THREAD

**Issue by HamsterHawk** [dangeredwolf/ModernDeck#318]
since it runs videos via the html viewer, all the videos play at max volume which half the time leads to an ear rape. please save the volume level after you close a video, and also playing the same volume level as the last video for every new video you play

**Comment by dangeredwolf** [comment]
Yeah this is annoying, I'll see what I can do to remedy this. Not sure if I can grab the volume from the existing player or if I'm going to have to write my own player...
